kolibri/plugins/coach/assets/src/views/plan/QuizSummaryPage/index.vue

```diff
@@ -255,6 +338,14 @@
             this.$store.dispatch('handleApiError', { error });
           });
       },
+      detailLink(learnerId) {
+        return this.classRoute(PageNames.REPORTS_QUIZ_LEARNER_PAGE_ROOT, {
+          learnerId,
+        });
+      },
+      exportCSV() {
+        this.$refs.table.exportCSV?.();
```

Comment:
I'm not sure that the `?.` will avoid the error here. I tried this out in the node repl. I think that this is a blocker unless it's fully expected for `exportCSV` to not be defined, in which case this can be rewritten to avoid the resulting errors in the console.

```javascript
> x = {a: 1}
{ a: 1 }
> x.a?.()
Uncaught TypeError: x.a is not a function
```

Reply:
Oh, no, no, the `?.()` is not supposed to be a validator to see if the `exportCSV` is a function, but rather that it is defined, and avoid breaking if its not. `exportCSV` is a very specific identifier and it is a verb, so having a non-function `exportCSV` attribute defined within the table ref component sounds like a very edge case. We could also write a couple of lines to make the validation validation more robust if needed, and just do
```javascript
if (typeof this.$refs.table.exportCSV === 'function') {
  this.$refs.table.exportCSV();
}
```

Reply:
At the end, I just updated this, just in case, to make the typeof validation. :open_hands:

Reply:
Right on, thanks Alex. That makes sense and this way avoids the possible error altogether.